manifests/params.pp

```diff
@@ -31,6 +31,10 @@
     $install_method = 'package'
     $package_name = 'app-crypt/certbot'
     $package_command = 'certbot'
+  } elsif $::osfamily == 'OpenBSD' {
```

Comment:
this whole block should probably be changed to a switch/case statement. You don't need to convert it, but feel free if you want to.

Reply:
Might also be a good candidate for hiera data files but that's probably out of scope for this PR.